gremlin-test/src/main/resources/org/apache/tinkerpop/gremlin/structure/io/graphml/tinkerpop-classic-normalized.xml

```diff
@@ -1,15 +1,18 @@
 <?xml version="1.0" ?>
 <graphml xmlns="http://graphml.graphdrawing.org/xmlns" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://graphml.graphdrawing.org/xmlns http://graphml.graphdrawing.org/xmlns/1.1/graphml.xsd">
     <key id="age" for="node" attr.name="age" attr.type="int"></key>
+    <key id="colorV" for="node" attr.name="color" attr.type="string"></key>
```

Comment:
This is a standard dataset. We can't change the data within it.

Reply:
Well. than perhaps rename it and place it aside?
The problem being solved is to add two properties on edge and vertex having the same name during run-time and serialize this new graph to GraphML.

Reply:
I think you'd missed my comment on the JIRA about this - please add a test here:

https://github.com/apache/tinkerpop/blob/e3018fb2e98df62a21272446f63cea5ee550408e/gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/structure/io/IoTest.java#L108

then , since it is a one-off test just for GraphML, I think you can just embed a small GraphML file as a String right in the test.